guides/graphite-cli/cli-command-reference.md

```diff
@@ -224,7 +238,7 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main.
+    * Don't prompt confirmation when Graphite suggests deleting a branch already-merged into main.
```

Comment:
Same question re:hyphenation of "already-merged"

Nit: let's be wordier in favor of readability by adding "which is"

"Don't prompt confirmation when Graphite suggests deleting a branch which is already merged into main."